db/migrate/20150112133542_create_announcements.rb

```diff
@@ -0,0 +1,14 @@
+class CreateAnnouncements < ActiveRecord::Migration
+  def change
+    create_table :announcements do |t|
```

Comment:
`course_announcements`. we have `instance_announcements` and `announcements` (global)